### DISCUSSION TITLE
Bug:  adding packages module provent YouTube YTDL-Core run

### DISCUSSION THREAD

**Discussion by NekoSuneVR** [eritislami/evobot#431] [Ideas]
**Describe the bug**
Comes to `Error: Error parsing info: Unable to retrieve video metadata`

**How To Reproduce**
1. installing node-fetch and discord.js-menu
2. Edited Help command have Pages then have it in 1 message for adding more commands on.
3. Made my own gaming data from custom .json path and read by command. (Genshin-impact-api or any API callouts breaks it too) 

**Expected behavior**
Just never loads up when install any other node packages from making a fork of BOT but adding own commands inside it

**Environment (add if possible)**
* Node.js version:  14.15.0

**Additional information & screenshots**

![image](https://user-images.githubusercontent.com/18028479/101123945-db6ff180-35ed-11eb-85cc-254d7011c7d9.png)

Main target doing this error, because packages installing issue. when try install package any NPM source it will not make YTDL-CORE RUN!

when default install it works fine. issue is in this stage preventing adding commands / modules to add in without YTDL-CORE Crashing.

and should move away from `YTDL-CORE-DISCORD` its Departed and its no longer able updates fix this issue because they github deleted source of files.

just now archive mode from last storage it saved.

**Comment by nullptr448** [comment]
I'm getting the exact same problem installing mongoose, this is a weird bug XD

**Comment by eritislami** [comment]
Don't create bug issues for programming support, converting to discussion.

**Reply by NekoSuneVR** [reply]
kinda is a bug, doesnt every time install node packages. do with something 1 files doesn't like install anything then what required.

like right now i can easy rebuild this without a issue with installing NPM packages but same time its 100% bug. if dont believe me check self 

install fresh copy this and install 1 random NPM packages and try run (prefix)play song

all you get is YTDL-Core issue form ya bot scripts

**Reply by NekoSuneVR** [reply]
only way i can void thing is run index.js with modules to other 2 folders of bot then install packages there.

thats only hard way get bot work while making own commands on bot. but takes more work todo

**Comment by Artistic18** [comment]
I am facing the same problem, can you please tell how to fix this?
